CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Overview
+To contribute to this project, follow the rules from the general [CONTRIBUTING.md](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md) document in the community repository.
+
+## Getting started
+Extending the contribution guidelines above here is how to quick start your contribution process.
+* The template is rendered using the AsyncAPI generator React renderer, [the authoring docs](https://github.com/asyncapi/generator/blob/master/docs/authoring.md#react-1) is great way to quick start your template journey to understand the setup.
+* The template support two examples generations which is used to inspect your changes. These can be generated using the scripts `npm run generate:examples`.
+* Do NOT commit the examples on your branch. Once your PR are merged the examples will be auto generated and committed automatically by the CI. This is to ease the PR review process.
```

Comment:
Sometimes generating examples will be great if you wanna show changes to reviewers. I think it should be included here with annotation that before merge it should be removed.

Reply:
good point 🤔

Reply:
@magicmatatjahu something like this?
```
* In most cases committing changes to the examples is a good thing, but try not to clutter up the PR. You may be asked to remove committed example code to remove clutter and ease the review process. Once your PR is merged the examples will be auto-generated and committed automatically by the CI. This is to ease the PR review process.
```

Reply:
@jonaslagoni Yes, it is good :) But add note that generated examples should be removed before merge - reviewers can miss about that.

Reply:
@magicmatatjahu done :)